hack/arktos-up.sh

```diff
@@ -18,6 +18,10 @@ PARTITION_CONFIG_DIR=${PARTITION_CONFIG_DIR:-"/var/run/kubernetes/"}
 
 VIRTLET_METADATA_DIR=${VIRTLET_METADATA_DIR:-"/var/lib/virtlet"}
 VIRTLET_LOG_DIR=${VIRTLET_LOG_DIR:-"/var/log/virtlet"}
+VIRTLET_DEPLOYMENT_FILES_DIR=${VIRTLET_DEPLOYMENT_FILES_DIR:-"/tmp"}
+
+# Default to current arktos-vm-runtime release-330 branch
+VIRTLET_DEPLOYMENT_FILES_SRC=${VIRTLET_DEPLOYMENT_FILES_SRC:-"https://raw.githubusercontent.com/futurewei-cloud/arktos-vm-runtime/release-330/deploy"}
```

Comment:
It's not suggested to use "release-330" as the version identifier. What will happen for next year's 330? :).  Consider using something like "release-0.5", "release-1.1", etc.

Reply:
Also the versioning of runtime could be separate from Arktos. They don't need to be always same.  For example, in the next several months Arktos version will increase. But arktos-vm-runtime version can stay same if there are no updates on it.

Reply:
it might be better we align with our docker image release number, which is currently 0.5.2.

Reply:
Aligning with docker image release number sounds a good idea to me.  All these (binary, image, configuration files) should share a single release number to avoid confusion.